frontend/src/pages/modelServing/screens/ServingRuntimeVersionStatus.tsx

```diff
@@ -0,0 +1,71 @@
+import { Alert, AlertProps, Label, Popover } from '@patternfly/react-core';
+import * as React from 'react';
```

Comment:
Super minor nitpick but the react line conventionally is the first import 
```suggestion
import * as React from 'react';
import { Alert, AlertProps, Label, Popover } from '@patternfly/react-core';
```